### DISCUSSION TITLE
How to configure Google Analytics 4?

### DISCUSSION THREAD

**Discussion by nvuillam** [squidfunk/mkdocs-material#2102] [Q&A] [answered]
I don't succeed to configure mkdocs.yml with Google analytics V4

In the doc and examples I found, GA Identifier starts with "UA" but the identifier I am provided by Google Analytics V4 starts with G-

Would it be possible to update documentation with : 
- how to configure with GA V4
- find the id in GA admin panel

My current configuration is visible here -> https://github.com/nvuillam/mega-linter/blob/master/mkdocs.yml

Thanks :)

**Comment by squidfunk** [comment, marked as answer]
Please see #2055 and #1972 for a discussion on `gtag.js`.
